.copywrite.hcl

```diff
@@ -0,0 +1,7 @@
+schema_version = 1
+
+project {
+  header_ignore = [
+    "dist/index.js"
```

Comment:
Can we ignore the whole dist directory

Reply:
We're encouraged to be very specific in what we ignore